common/legacy/class.LegacyAutoLoader.php

```diff
@@ -133,6 +141,36 @@ public function autoload($pClassName)
                 return;
             }
         }
+
+        $this->loadFromRootExtension($tokens);
+    }
+
+    private function loadFromRootExtension(array $classNameTokens): void
```

Comment:
What do you think about creating a test case to cover this new part? So that we can avoid having similar issues in the future. :)

Reply:
This legacy autoloader should be removed at some point.
I guess it would make more sense to comment it as "do not edit" :P